### PR TITLE
Prep for version 2.3.1

### PR DESCRIPTION
**Pull request by beylmk**
- Bumped purchases-android to 4.3.1 [Changelog here](https://github.com/RevenueCat/purchases-android/releases/4.3.1), 
which fixes canMakePayments not returning (see related issue: https://github.com/RevenueCat/purchases-unity/issues/61)
- Bumped purchases-ios to 3.12.3 [Changelog here](https://github.com/RevenueCat/purchases-ios/releases/tag/3.12.3)
- Bumped purchases-hybrid-common to 1.9.1 [Changelog here](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/1.9.1)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 2.3.1
+
+- Bumped purchases-android to 4.3.1 [Changelog here](https://github.com/RevenueCat/purchases-android/releases/4.3.1), 
+which fixes canMakePayments not returning (see related issue: https://github.com/RevenueCat/purchases-unity/issues/61)
+- Bumped purchases-ios to 3.12.3 [Changelog here](https://github.com/RevenueCat/purchases-ios/releases/tag/3.12.3)
+- Bumped purchases-hybrid-common to 1.9.1 [Changelog here](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/1.9.1)
+
+
```
Comment (vegaro) [inline CHANGELOG.md:8]: Extra space

File: RELEASING.md [hunk 2]
```diff
@@ -8,4 +8,4 @@
 1. `git push origin main && git push --tags`
 1. Create a new release in github and upload
 1. `npm publish`
-1. Update the version in `package.json` and `plugin.xml` to x.y.z-SNAPSHOT, with x.y.z being the next version
+1. Update the version in `package.json` and `plugin.xml` to x.y.z-SNAPSHOT, with x.y.z being the next version
```
Comment (vegaro) [inline RELEASING.md:11]: what changed here?

File: src/android/PurchasesPlugin.java [hunk 3]
```diff
@@ -33,7 +33,7 @@
 public class PurchasesPlugin extends AnnotatedCordovaPlugin {
 
     public static final String PLATFORM_NAME = "cordova";
-    public static final String PLUGIN_VERSION = "2.3.0";
+    public static final String PLUGIN_VERSION = "1";
```
Comment (vegaro) [inline src/android/PurchasesPlugin.java:36]: 😄

**Review by vegaro — CHANGES_REQUESTED** [review]

File: RELEASING.md (hunk 2 above)
Comment (beylmk) [reply RELEASING.md:11]: honestly? no clue

**Review by vegaro — APPROVED** [review]

**Pull request merged**